examples/build.sh

```diff
@@ -28,20 +28,31 @@ cd "$BAL_HOME_DIR" &&
   bal push --repository=local
 
 # Remove the cache directories in the repositories
-cacheDirs=($(ls -d "$BAL_CENTRAL_DIR"/cache-* 2>/dev/null))
+cacheDirs=$(ls -d $BAL_CENTRAL_DIR/cache-* 2>/dev/null) || true
 for dir in "${cacheDirs[@]}"; do
   [ -d "$dir" ] && rm -r "$dir"
 done
 echo "Successfully cleaned the cache directories"
 
+# Create the package directory in the central repository, this will not be present if no modules are pulled
+mkdir -p "$BAL_CENTRAL_DIR/bala/ballerinax/$BAL_PACKAGE_NAME"
+
 # Update the central repository
 BAL_DESTINATION_DIR="$HOME/.ballerina/repositories/central.ballerina.io/bala/ballerinax/$BAL_PACKAGE_NAME"
 BAL_SOURCE_DIR="$HOME/.ballerina/repositories/local/bala/ballerinax/$BAL_PACKAGE_NAME"
 [ -d "$BAL_DESTINATION_DIR" ] && rm -r "$BAL_DESTINATION_DIR"
 [ -d "$BAL_SOURCE_DIR" ] && cp -r "$BAL_SOURCE_DIR" "$BAL_DESTINATION_DIR"
 echo "Successfully updated the local central repositories"
 
+echo "$BAL_DESTINATION_DIR"
+echo "$BAL_SOURCE_DIR"
+
 # Loop through examples in the examples directory
 find "$BAL_EXAMPLES_DIR" -type f -name "*.bal" | while read -r BAL_EXAMPLE_FILE; do
   bal "$BAL_CMD" --offline "$BAL_EXAMPLE_FILE"
 done
+
+# Remove generated JAR files
+find "$BAL_HOME_DIR" -maxdepth 1 -type f -name "*.jar" | while read -r JAR_FILE; do
+  rm "$JAR_FILE"
+done
```

Comment:
Missing EoF newline

Reply:
Addressed in [a10fbf8](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/a10fbf83fa4673cdc6e33ae0f116932ea4369f49).